client/src/Components/Townbox/index.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/prop-types */
+import React from 'react'
+import './index.css'
+import {Button} from '../Button'
+
+export interface TownboxProps {
+  /**
+   * Header colour: enter either bg-orange or bg-primary
+   */
+  headerColor?: string
```

Comment:
If this should only work for these two values the typescript type should specify them. Also I feel a nicer input for the props might be to just have `headerColor: 'orange'` and then inside the component having: `bg-${headerColor}`. This also has the benefit of stopping people from using this prop to add any css rules they want to the component.